include/zephyr/bluetooth/audio/bap.h

```diff
@@ -509,12 +588,13 @@ struct bt_bap_unicast_server_cb {
 	 * @param[out] stream  Pointer to stream that will be configured for the endpoint.
 	 * @param[out] pref    Pointer to a QoS preference object that shall be populated with
 	 *                     values. Invalid values will reject the codec configuration request.
+	 * @param[out] rsp     Object for the ASE operation response.
```

Comment:
Should we add something like
```suggestion
	 * @param[out] rsp     Object for the ASE operation response. Only used if the return value is non-zero. 
```